profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.admin.inc

```diff
@@ -26,6 +43,52 @@ function nexteuropa_varnish_admin_settings_form($form, $form_state) {
 }
 
 /**
+ * Redirect the user from "General" page to the "purge all" confirmation form.
+ */
+function nexteuropa_varnish_purge_all_confirm($form, &$form_state) {
+  $form_state['redirect'] = 'admin/config/system/nexteuropa-varnish/purge_all';
+}
+
+/**
+ * Generates the form allowing triggering the full cache purge.
+ */
+function nexteuropa_varnish_purge_all_form($form, &$form_state) {
+  $description = t("The action you are about to perform has deep impacts on site's performances!");
```

Comment:
I think its better: "The action you are about to perform has a deep impact on the site's performance!"